polls/views.py

```diff
@@ -159,20 +152,23 @@ def test_func(self):
         return self.request.user.is_superuser
 
 
-class AdminAPIView(APIView):
-    authentication_classes = [TokenAuthentication]
+class ThanksView(TemplateView):
+    template_name = "polls/thanks.html"
 
+
+class AdminManagementView(LoginRequiredMixin, UserPassesTestMixin, View):
     def get(self, request):
-        return Response({"data": "hello"}, status.HTTP_200_OK)
+        annotators = get_user_model().objects.exclude(is_superuser=True)
```

Comment:
Shouldn't these be AnnotatorProfiles instead of Users?

Reply:
Annotator profile is a secondary profile for storing additional values, yes we can use it too, but filtering batches and tasks would require more queries than supposed